javascript/chrome-debugging

```diff
@@ -0,0 +1,32 @@
+# How to use chrome debugger 
+
+Chrome developer tools are a wonderful way to see what is exactly happening with the code at every iteration. 
+
+### Instructions
+Make a simple html file with Notepad by copying this inside it and saving as .html
+``` 
+<!DOCTYPE html>
+<html>
+<head>
+<title>Hello, HTML!</title>
+<script>
+
+Your code goes here!
+
+</script>
+</head>
+<body>
+<h1>Let's Count to 10 with JavaScript!</h1>
+<p id="theCount"></p>
+</body>
+</html>
+``` 
+![Chrome debugger layout](https://drive.google.com/open?id=0B3Ws6ZqKKpLOT19qU01TTW1SeVk)	
+Open your .html file in chrome and 	from control tools --> more tools choose Developer tools
+
+Click Sources from the top menu, select your file name and refresh page just in case.  
+At the upper right side of the screen there is a dot with an arrow over it, when you start clicking it, the debugger goes through the code line by line and shows every single iteration next to the code and on the log window on the left. 
+
+To start debugging from the top refresh the page.
+
+Keep in mind to call the function at the end of the code with a test parameter otherwise the debugger won't run.
```

Comment:
It might also be worth lining to the more detailed readme on debugging: http://codingforeveryone.foundersandcoders.org/programmer-skills/Debugging-Javascript.html